packages/sit-onyx/src/components/OnyxLink/types.ts

```diff
@@ -0,0 +1,24 @@
+import type { TextSize } from "../../types/fonts";
+
+export type OnyxLinkProps = {
+  /**
+   * URL that the link points to.
+   */
+  href: string;
+  /**
+   * Where to display the linked URL (same tab, new tab etc.).
+   * For `_blank`, the `rel="noopener noreferrer"` will be set automatically.
+   */
+  target?: LinkTarget;
+  /**
+   * Text size.
+   */
+  size?: TextSize;
+  /**
+   * If `true`, the monospace font family will be used instead of the default one.
+   */
+  monospace?: boolean;
```

Comment:
We shouldn't use properties for styling. The component will mainly be used inline in running text, where the font styling should come from the surrounding text. With this approach, the user would need to always manually sync text block style and link style.

Reply:
Done